components/table/table-wrapper.js

```diff
@@ -243,11 +245,16 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-border-radius: 0.3rem;
 				--d2l-table-border-radius-sticky-offset: calc(1px - var(--d2l-table-border-radius));
 				--d2l-table-cell-height: 41px; /* min-height to be 62px including border */
-				--d2l-table-cell-padding: 0.5rem 1rem;
+				--d2l-table-cell-padding: 0.5rem 15px;
```

Comment:
Would we not want to use this padding within the table-col-sort component as well? 

Also we'd likely also want to use rem for this as well (`0.75 rem` for this case; it's generally spec px value / 20 px)